sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -866,6 +866,7 @@
                 -snippetpath ${project.basedir}/sdk/storage/azure-storage-queue/src/samples/java
                 -snippetpath ${project.basedir}/sdk/textanalytics/azure-ai-textanalytics/src/samples/java
               </additionalOptions>
+              <aggregate>true</aggregate>
```

Comment:
Is this a configuration for the Javadoc plugin or a report configuration? I don't see an option for `aggregate` in the Javadoc plugin documentation

Reply:
This change is not required and is reverted.